frontend/test/pull-card-demo.tsx

```diff
@@ -75,4 +77,8 @@ function Row({ title, pullDatas }: { title: string; pullDatas: PullData[] }) {
   );
 }
 
+Row.defaultProps = {
+  linesChanged: false,
```

Comment:
Given that the default is falsy I don't think we need a default as null and undefined are both falsy.